packages/react-frontend/src/routes/AlbumInfo.jsx

```diff
@@ -126,7 +99,7 @@ export default function AlbumInfo() {
               {album.release_date.split("-")[0] + " "}
               {artist.artist_name} |{"  "}
               <span className="rating-stars">
-                {starRating}{" "}
+                <Rating name="album-rating" value={parseFloat(avgRating)} precision={0.5}/>
```

Comment:
Hey looking at the materials UI documentation. The precision could be less than 0.5. Like 0.1 for example and then the stars would more accurately display the result you're looking for. Keeping it at 0.5 only makes the stars display in half amount of increments. https://mui.com/material-ui/react-rating/